packages/mui-material/src/Button/Button.js

```diff
@@ -279,47 +280,28 @@ const ButtonEndIcon = styled('span', {
 }));
 
 const Button = React.forwardRef(function Button(inProps, ref) {
-  const props = useThemeProps({ props: inProps, name: 'MuiButton' });
-  const {
-    className: classNameContext,
-    color: colorContext,
-    disabled: disabledContext,
-    disableElevation: disableElevationContext,
-    disableFocusRipple: disableFocusRippleContext,
-    disableRipple: disableRippleContext,
-    fullWidth: fullWidthContext,
-    size: sizeContext,
-    variant: variantContext,
-  } = React.useContext(ButtonGroupContext);
+  // props priority: `inProps` > `contextProps` > `themeDefaultProps`
+  const contextProps = React.useContext(ButtonGroupContext);
+  const mergedProps = mergeProps(contextProps, inProps);
+  const props = useThemeProps({ props: mergedProps, name: 'MuiButton' });
   const {
     children,
-    className,
-    color: colorProp,
+    color = 'primary',
     component = 'button',
-    disabled: disabledProp,
-    disableElevation: disableElevationProp,
-    disableFocusRipple: disableFocusRippleProp,
-    disableRipple: disableRippleProp,
+    className,
+    disabled = false,
+    disableElevation = false,
+    disableFocusRipple = false,
```

Comment:
```suggestion
    disableFocusRipple = false,
    disableRipple = false,
```
I can't see why this was removed

Reply:
It was introduced in the context api PR which causes the regression.

Reply:
Ah got it! Ok